code/modules/mob/living/carbon/human/species.dm

```diff
@@ -141,6 +141,10 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	/// The body temperature limit the body can take before it starts taking damage from cold.
 	var/bodytemp_cold_damage_limit = BODYTEMP_COLD_DAMAGE_LIMIT
 
+	//the species that body parts are surgically compatible with
```

Comment:
```suggestion
	/// the species that body parts are surgically compatible with
```

`///` == doc comment.

Also, link the file where people can find the defines, or even explain them here as well.

Reply:
fixed